modules/features2d/include/opencv2/features2d.hpp

```diff
@@ -1580,6 +1587,104 @@ class CV_EXPORTS_W BOWImgDescriptorExtractor
 
 //! @} features2d_category
 
+/****************************************************************************************\
+*                                         Annoy                                          *
+\****************************************************************************************/
+
+//! @addtogroup features2d_annoy
+//! @{
+
+class CV_EXPORTS_W AnnoyIndex
+{
+public:
+    /** @brief Metrics used to calculate the distance between two feature vectors.
+     */
+    enum Distance
+    {
+        ANNOY_DIST_EUCLIDEAN,
```

Comment:
ANNOY_DIST is inside AnnoyIndex, don't duplicate prefix => ANNIndex::DIST_EUCLIDEAN.